kbc.schema.json

```diff
@@ -0,0 +1,54 @@
+{
+  "$schema":"https://json-schema.org/draft/2020-12/schema",
```

Comment:
I would prefer we used this version everywhere, just noting that the other schemas use draft 7.

I'd recommend keeping this and let's take an action to update the other schemas.

Reply:
We also need to make sure our libs are set up to handle 2020-12

Reply:
lib support for `2020-12` is largely untested at the moment. we know for a fact that 07 works. if we can get someone with cycles to test `2020-12` across several languages or even just rust, js, go, and dart that'd be awesome